src/api/Stilus.php

```diff
@@ -101,15 +101,17 @@ public function main(): void
         }
 
         $server = null;
-        if (isset($config['api']) &&
-            isset($config['api']['http_server']) &&
-            isset($config['api']['http_server']['enable']) &&
-            $config['api']['http_server']['enable']
-        ) {
+        if ($this->isServerEnable($config))
```

Comment:
@bluebat-pl If you need it to be more expressive please use variable: `$isServerEnabled = ....` instead method call.